ordering/containers/OrderCreator.js

```diff
@@ -124,8 +124,15 @@ function OrderCreatorContainer (props) {
     actions.orders.create(values)
   }
 
+  const hasNoCurrentAgentGroupProfiles = isEmpty(props.currentAgentGroupProfiles)
+
   const nextProps = merge(props, {
-    onSubmit: handleSubmit
+    onSubmit: handleSubmit,
+    hasNoCurrentAgentGroupProfiles,
+    initialValues: {
+      supplierAgentId: hasNoCurrentAgentGroupProfiles ? 'NEW_SUPPLIER' : '',
```

Comment:
won't this always set the initial values to new supplier? since i think this code will always run first without having loaded the agents, which will permanently set the initial values.

it's probably fine to always default this to new supplier though, at least worth trying as a new test on users.